src/main/java/dev/openfeature/javasdk/EvaluationContext.java

```diff
@@ -103,4 +110,49 @@ public static EvaluationContext merge(EvaluationContext ctx1, EvaluationContext
 
         return ec;
     }
+
+    public EvaluationContext fromMap(Map<String, Object> map) {
+        EvaluationContext context = new EvaluationContext();
+        context.integerAttributes.putAll(
+                map.entrySet()
+                        .stream()
+                        .filter(entry -> entry.getValue() instanceof Integer)
+                        .collect(Collectors.toMap(Map.Entry::getKey, e -> (Integer) e.getValue()))
+        );
+        context.stringAttributes.putAll(
+            map.entrySet()
+                .stream()
+                .filter(entry -> entry.getValue() instanceof String)
+                .collect(Collectors.toMap(Map.Entry::getKey, e -> (String)e.getValue()))
+        );
+        context.booleanAttributes.putAll(
+            map.entrySet()
+                .stream()
+                .filter(entry -> entry.getValue() instanceof Boolean)
+                .collect(Collectors.toMap(Map.Entry::getKey, e -> (Boolean)e.getValue()))
+        );
+//        Hmmm, this won't work as we already have a string type.
+//        I would recommend changing the type to Map<String, JsonNode> from Jackson
```

Comment:
I'm a jackson newbie, but my understanding is that jsonnode's are an immutable json tree structure. That seems fine to me for this.